### ISSUE TITLE
Using the code from the official api documentation, I cannot pass the feature column to the keras model

### ISSUE THREAD

**Issue by anavanab99** [tensorflow/tensorflow#39485]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://tensorflow.google.cn/versions/r2.1/api_docs/python/tf/io/parse_example
https://tensorflow.google.cn/versions/r2.1/api_docs/python/tf/feature_column/bucketized_column?hl=en
## Description of issue (what needs changing):
but the code ：features = tf.io.parse_example (
serialized = record, features = tf.feature_column.make_parse_example_spec (l_list))
In the case of serialized = record, what type of object should be filled here to complete parse_example?

### Clear description

For example, why should someone use this method? How is it useful?

### Correct links

Is the link to the source code correct?

### Parameters defined

Are all parameters defined and formatted correctly?

### Returns defined

Are return values defined?

### Raises listed and defined

Are the errors defined? For example,
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/feature_column/categorical_column_with_vocabulary_file#raises

### Usage example
dnn_activation='relu'
volume = tf.feature_column.numeric_column('volume', shape=number_shape)
l_list=[volume]
features = tf.io.parse_example(
serialized=record, features=tf.feature_column.make_parse_example_spec(l_list))
input_l=keras.layers.DenseFeatures(l_list)
dnn1=keras.layers.Dense(300,activation=dnn_activation)(input_l)
output=keras.layers.Dense(1,activation='tanh')(dnn1)
model = Model(inputs = [input_l],outputs = [output])
model.compile(loss = "mae",optimizer = "Adam")

I tried to input the feature column to keras model, but the code ：features = tf.io.parse_example (
serialized = record, features = tf.feature_column.make_parse_example_spec (l_list))
In the case of serialized = record, what type of object should be filled here to complete parse_example? I studied the official API carefully, but the official API filled in serialized = ... instead. I can't understand how to complete the code. I tried to fill in various methods such as tfrecord, fill in tensor, and I couldn't complete the code. My data is a csv file. Can you provide a complete example, including data reading, feature column processing, and training?? I really cannot complete the code. Thank you very much, this is very important to me. Thank you very much, my email: 402868327@qq.com
Is there a usage example?

See the API guide: https://www.tensorflow.org/community/contribute/docs_ref
on how to write testable usage examples.

### Request visuals, if applicable

Are there currently visuals? If not, will it clarify the content?

### Submit a pull request?

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by amahendrakar** [comment]
@anavanab99,
The documentation for all the APIs generally have a usage example with them.

- [tf.io.parse_example](https://www.tensorflow.org/api_docs/python/tf/io/parse_example) - [Used in the tutorial](https://www.tensorflow.org/guide/data#parsing_tfexample_protocol_buffer_messages)

 - [tf.feature_column.bucketized_column](https://www.tensorflow.org/api_docs/python/tf/feature_column/bucketized_column) - [Used in the tutorial](https://www.tensorflow.org/tutorials/structured_data/feature_columns#choose_which_columns_to_use)

Please check the above links for the APIs and their respective usage example and let us know if it helps. Thanks!

**Comment by amahendrakar** [comment]
Any updates regarding this issue? Thanks!

**Comment by anavanab99** [comment]
> Any updates regarding this issue? Thanks!

I'm very sorry, because the recent initial difference has been unable to reply to this problem, I tried to solve one of the problems with other methods, but tf.feature_column.sequence_numeric_column, such feature columns can still not be input to the keras model. Simple and incomplete, unable to understand the use of api, dear google staff, do you consider a more complete api documentation description? In addition, this is my error code.
[tf_sequence_numeric_column_keras.txt](https://github.com/tensorflow/tensorflow/files/4693618/tf_sequence_numeric_column_keras.txt)

**Comment by anavanab99** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you.

The tensorflow API documentation on featurecolumns is too simple and incomplete, and we really cannot understand its detailed usage. However, feature engineering is indeed very important for the network.

**Comment by nikitamaia** [comment]
Hi @anavanab99, trying to better understand the request here. Are you looking for a usage example of `tf.feature_column.sequence_numeric_column` ?

Also linking to the Keras tutorials [Classify structured data with feature column](https://www.tensorflow.org/tutorials/structured_data/feature_columns)s and[ Classify structured data using Keras Preprocessing Layers](https://www.tensorflow.org/tutorials/structured_data/preprocessing_layers). If there are specific things you think are missing, please let us know.
